Dockerfile

```diff
@@ -1,13 +1,10 @@
-FROM registry.centos.org/centos/centos:7
+FROM registry.access.redhat.com/ubi8/python-36:latest
```

Comment:
I've used [ubi-minimal](https://github.com/fabric8-analytics/fabric8-analytics-server/blob/master/Dockerfile) in the api-server and other repos where I migrated to ubi due to the size differences.

I just want to know is it still applicable.